layouts/onboarding/list.html

```diff
@@ -286,7 +286,34 @@ <h3 class="mt-3">Open Science</h3>
             The UNESCO Open Science Recommendations are a helpful resource to begin your journey. At the SciLifeLab Data Centre, resources
             such as the Figshare data repository are helpful for sharing your research data and other outputs.
         </p>
-        
+
+        <!--Open science box -->
+        <div style="background-color: #D2E5E7;">
+            <div class="row align-items-center p-3">
+                <div class="col-md-8 ps-4">
+                    <h3>The Open Science Team</h3>
+                    <p>
+                        The Open Science team at SciLifeLab coordinates and supports the integration of Open Science and FAIR principles
+                        across the organisation. The team works with colleagues across SciLifeLab and with Open Science initiatives
+                        nationally and internationally to develop guidance, tools, and foster collaboration through communities of practice.
+                        In doing so, the team helps embed Open Science practices in all research at SciLifeLab and its affiliated universities.
+                        For more information visit the
+                        <a href="http://data.scilifelab.se/open_science">Open Science pages</a>.
```

Comment:
Better to use relative URL (i.e. just `/open_science/`) for internal pages. In the end its all the same, but using relative URLs can be beneficial in some cases.

For example, if we decide to change our domain name to something else, we don't have to worry about fixing this again, etc